adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostBannerAd.m

```diff
@@ -101,26 +100,19 @@ - (void)getBannerWithSize:(GADAdSize)adSize {
                         return;
                       }
 
-                      if (!success) {
+                      if (cbError) {
                         NSError *error = GADMAdapterChartboostErrorWithCodeAndDescription(
                             GADMAdapterChartboostErrorInitializationFailure,
                             @"Failed to initialize Chartboost SDK.");
                         [strongConnector adapter:strongAdapter didFailAd:error];
                         return;
                       }
 
-                      GADMChartboostExtras *extras = [strongConnector networkExtras];
-                      if (extras) {
-                        [Chartboost setFramework:extras.framework
-                                     withVersion:extras.frameworkVersion];
-                      }
-
                       CHBMediation *mediation = GADMAdapterChartboostMediation();
                       strongSelf->_bannerAd = [[CHBBanner alloc] initWithSize:chartboostAdSize
                                                                      location:adLocation
                                                                     mediation:mediation
                                                                      delegate:strongSelf];
-                      strongSelf->_bannerAd.automaticallyRefreshesContent = NO;
```

Comment:
Can you clarify how the banner refresh is now handled?

Reply:
Chartboost does not support automatic refresh anymore.